test/ncengine/window/NcWindowImpl.inl

```diff
@@ -0,0 +1,69 @@
+#include "window/Window.h"
```

Comment:
We don't rely directly on the symbol `NcWindowImpl` in graphics, correct? I'd bet its just the base `NcWindow`. If true, this file and type should be named something like `NcWindowStub`.